app/api/wallet/network.ts

```diff
@@ -24,3 +24,25 @@ export function getBip32Option (envNetwork: EnvironmentNetwork): Bip32Options {
     wif: network.wifPrefix
   }
 }
+
+export function getTxURLByNetwork (network: EnvironmentNetwork, txId: string): string {
```

Comment:
don't put it here, this is strictly for wallet implementation

Reply:
should implement this at context level via:

```ts
const { getTransactionUrl } = useDeFiScan()
const url = getTransactionUrl(txid)
```

Reply:
added [DeFiScan in context hook](https://github.com/DeFiCh/wallet/pull/404/files#diff-220a25fc8649f07414f4522acbf078ebccc8cbe3f062c4ae1ce65df06e7dc467)

let me know if i missed out anything